news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "The First osu! Manga: osu! Combat Championship"
+date: 2023-10-21 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted.
+
+![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)
+
+**Calling all osu! enthusiasts!**
+ 
+Prepare yourselves for a journey like none other, as we introduce the highly anticipated **osu! Combat Championship** (OCC) manga!
+
+ Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.
+
+For those eager to dive into the manga and skip the introductions, simply click the button below to head over to Mangadex and start reading.
+
+<p align="center">
+  <a href="https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship">
+    <img src="/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/occbuttonnew.png" alt="Character Panels">
+  </a>
+</p>
+
+Curious about which of your favorite players are in The osu! Combat Championship manga? Check out the character introductions below:
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+WhiteCat: WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harbored blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```

Comment:
I am not sure if the `player:` thing is necessary if you have the description under the banner and it mentions them by name
```suggestion
WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harboured blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```